src/components/theme.ts

```diff
@@ -0,0 +1,270 @@
+import { css } from '@emotion/react';
+
+/**
+ * @type {Object.<string, string>}
+ * @property {string} fontsize returns px value
+ */
+const fontSize = {
```

Comment:
Having `src/components/theme.ts` is great but just something to keep in mind that this may all be replaced by the design system. For now, it's ok to keep it since we could use some parts of this.

Reply:
Yeah I figured a lot of this will be changed out, just wanted something for now lol.